data-prepper-api/src/main/java/org/opensearch/dataprepper/model/event/Event.java

```diff
@@ -135,9 +143,15 @@ public interface Event extends Serializable {
 
     JsonStringBuilder jsonBuilder();
 
-    public abstract class JsonStringBuilder {
+    abstract class JsonStringBuilder {
```

Comment:
Thanks for making this improvement - inner classes on interfaces are automatically public, so no keyword is necessary.